mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping

```diff
@@ -1 +1,25 @@
 CLASS net/minecraft/class_556 net/minecraft/client/render/item/HeldItemRenderer
+	FIELD field_2401 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2402 stack Lnet/minecraft/class_31;
+	FIELD field_2403 height F
+	FIELD field_2404 prevHeight F
+	FIELD field_2405 blockRenderManager Lnet/minecraft/class_13;
+	FIELD field_2406 itemRenderer Lnet/minecraft/class_321;
```

Comment:
Might wanna name this mapItemRenderer

Reply:
https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/client/render/MapRenderer.MapTexture.html
ty dany